apps/web/src/components/OverviewRow.tsx

```diff
@@ -29,26 +31,37 @@ export const OverviewRow = ({
     0,
     Math.min(4, formInstances.length),
   );
+
   return (
     <>
       <Box w={rowWidth}>
         <Flex justifyContent="space-between" alignItems="center">
           <Flex alignItems="center">
-            <Heading as="h2">{title}</Heading>
-            <Flex
-              marginLeft="13px"
-              backgroundColor={color}
-              height="18px"
-              width="32px"
-              borderRadius="12"
-              justifyContent="center"
-              alignItems="center"
-            >
-              <Text fontSize="14px" fontWeight="700" color="#756160">
-                {formInstances.length}
-              </Text>
-            </Flex>
+            <Heading as="h2">
+              {title == 'To Do'
+                ? `You have ${formInstances.length} forms waiting for you.`
+                : title}
+            </Heading>
+
+            {title != 'To Do' ? (
+              <Flex
+                marginLeft="13px"
+                backgroundColor={color}
+                height="18px"
+                width="32px"
+                borderRadius="12"
+                justifyContent="center"
+                alignItems="center"
+              >
+                <Text fontSize="14px" fontWeight="700" color="#756160">
+                  {formInstances.length}
+                </Text>
+              </Flex>
+            ) : (
+              <></>
+            )}
           </Flex>
+
           <Link href={link}>
             <Flex alignItems="center">
               <Text fontWeight="500" fontSize="16px" color="#4C658A">
```

Comment:
Can we change the title to `To-do`? Since we don't need to use it in the actual "title" of the overview row and "To-do" is how it's displayed in the Figma.